Cargo.toml

```diff
@@ -9,7 +9,11 @@ license = "Apache-2.0"
 
 [features]
 default = ["tokio-net"]
-rustls = ["tokio-rustls"]
+rustls-core = ["tokio-rustls"]
+rustls-aws-lc = ["rustls-core", "tokio-rustls/aws-lc-rs"]
+rustls-fips = ["rustls-aws-lc", "tokio-rustls/fips"]
+rustls-ring = ["rustls-core", "tokio-rustls/ring"]
+rustls = ["rustls-aws-lc", "tokio-rustls/default"]
```

Comment:
This should probably also include "tokio-rustls/tls12" and "tokio-rustls/logging" since those are included in the default features.

Reply:
But aren't we including the `tokio-rustls/default` feature already? It should pull them in.

Reply:
Oh, yeah you're right